### ISSUE TITLE
test failure with new lubridate

### ISSUE THREAD

**Issue by vspinu** [rOpenGov/sweidnumbr#102]
Most recent revdeps check on lubridate showed the following error:

```
checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  sweidnumbr: R tools to handle swedish identity numbers.
  https://github.com/rOpenGov/sweidnumbr
  
  1. Failure (at test-pin_to_date.R#6): pin_to_date ------------------------------
  pin_to_date(pin = c("196408233234", "186408833224")) inherits from Date not POSIXct
  
  testthat results ================================================================
  OK: 175 SKIPPED: 0 FAILED: 1
  1. Failure (at test-pin_to_date.R#6): pin_to_date 
  
  Error: testthat unit tests failed
  In addition: There were 12 warnings (use warnings() to see them)
  Execution halted
```

I am not completely sure, but it seems to be related to the recent change in the output of `ymd` family of functions in lubridate https://github.com/hadley/lubridate/issues/391

Those function now return Date objects unless tz= argument is non-NULL. The fix is either to supply tz="UTC" argument to mdy or to explicitly convert with as.POSIXct.

**Comment by MansMeg** [comment]
Thanks!

Maybe we should change the test suite so that it returns a Date object instead? Or is there any reasons to prefer POSIXct? This function is simply just to handle dates (and calculate age for swedish id numbers)?

**Comment by MansMeg** [comment]
By the way has the new version of lubridate been published on CRAN?

**Comment by vspinu** [comment]
> Or is there any reasons to prefer POSIXct?

No particular reason. The change has been made for consistency reasons mostly. All lubridate functions should work transparently on dates and posixct. I think in your case your pin_to_date function expects posixct, right? So maybe the right thing is to generalize it to Date as well.

> By the way has the new version of lubridate been published on CRAN?

I have just uploaded it. It has to go through the review first. Given that there have been such errors in revdeps I had to notify the authors first.
